frontend/wallet/src/render/views/onboarding/AccountCreationStep.re

```diff
@@ -29,16 +35,36 @@ module Styles = {
       marginLeft(`px(80)),
     ]);
   };
-
+  let header = {
+    merge([
+      Theme.Text.Header.h1,
+      style([animation(fadeIn, ~duration=1000, ~iterationCount=`count(1))]),
+    ]);
+  };
   let heroBody = {
     merge([
-      Theme.Text.Body.regular,
-      style([maxWidth(`rem(21.5)), color(Theme.Colors.midnightBlue)]),
+      Theme.Text.Body.regularLight,
+      style([
+        opacity(0.),
+        maxWidth(`rem(21.5)),
+        color(Theme.Colors.midnightBlue),
+        animation(fadeIn, ~duration=1050, ~iterationCount=`count(1)),
+        animationDelay(250),
+        animationFillMode(`forwards),
+      ]),
     ]);
   };
   let buttonRow = {
     style([display(`flex), flexDirection(`row)]);
   };
+  let textFields = {
+    style([
+      opacity(0.),
+      animation(fadeIn, ~duration=1050, ~iterationCount=`count(1)),
```

Comment:
Maybe bring the duration down to something a little snappier, I find that animation durations around 1s feel a little slow. Maybe try something in the 200-600ms range.

Reply:
Changed durations to 500ms, and for Setup Complete, only the text fades in. I think having the heading there without fading in seems "reward"-like